What is it like being a teacher page (#4405)
DFE-Digital/get-into-teaching-app: app/views/content/a-day-in-the-life-of-a-teacher.md

---

>   title: "What it's like to be a teacher"

Reviewer: We will need to add this page to the navigation and into one of the life as a teacher sections. Perhaps either teaching as a career?

    - everyday
    - timetable
  quote:
    q-gavin-ict:
      text: "Getting involved with extracurricular activities can be extremely rewarding."
      name: "Gavin, assistant headteacher"
      classes: quote--indent
    q-jane:
      text: "It’s all about the relationships you build with pupils. If they know they’ll ‘get away’ with disruptive learning, they will disrupt learning!"
      name: "Jane, teaching adviser"
      classes: quote--indent
    q-sarah-geography:
      text: "I sometimes do a couple of hours at the weekend but I try to keep my weekends free to make sure I have some real downtime."
      name: "Sarah, geography teacher"
      classes: quote--indent
    q-helen-food:

>       text: "It's not an easy career. There will be challenges along the way but persevere and the rewards are amazing. It's my 35th year and every day is still different. Do not expect too much of yourself too soon - 1 day at a time but you will make a difference and you will be remembered."

Reviewer: ```suggestion
      text: "It's not an easy career. There will be challenges along the way but persevere and the rewards are amazing. It's my 35th year and every day is still different. Do not expect too much of yourself too soon. You'll make a difference and be remembered."
```
I didn't think this quote flowed quite right - I've tried to make it a bit smoother but my suggestion might not be the right solution.

      name: "Helen, food tech teacher"
      classes: quote--indent
    q-phil-secondary:
      text: "Seeing the students arrive and knowing they can’t wait to get to my class. There’s no feeling like it."
      name: "Phil, secondary teacher"
      classes: quote--indent
    q-jessica-science:
      text: "There’s nothing better than those light bulb moments when students just get it."
      name: "Jessica, head of science"
      classes: quote--indent
    q-richard-art:
      text: "The fact that I continue to love the subject I teach has meant that I have never grown weary of doing what I do."

>       name: "Richard, art teacher"

Reviewer: Love that we now have quotes from art teachers!

      classes: quote--indent
    q-mike-secondary:

>       text: "If you’ve had a hard day, there’s always someone there to listen - even if it’s just a two-minute natter or moan."

Reviewer: ```suggestion
      text: "If you’ve had a hard day, there’s always someone there to listen - even if it’s just a 2 minute natter or moan."
```

      name: "Mike, secondary teacher"
      classes: quote--indent
    q-aaron-science:
      text: "Teaching offers you something few other careers can offer you. To make a real difference to a large number of students lives for the better."
      name: "Aaron, science teacher"
      classes: quote--indent

---
Teaching is a challenging but rewarding career that allows you to make a significant impact on pupils' lives. With no two days being the same, it’s an excellent choice for those who thrive in a dynamic and varied work environment.

$q-richard-art$

## A day in the life of a teacher

> 

Reviewer: We say above that no day is the same - is it worth adding a note that this is an example of what teaching could be like?

### Morning

#### 7:30am arrive at school

Teachers often start their day early, arriving at school somwhere between 7:00am to 8:00am. The morning is usually spent preparing resources for the day ahead so that everything is organised for when the pupils arrive.

#### 8:45am pupils arrive

> Depending on your role, you might be responsible for a form group or registering students. During this time, pupils typically engage in morning activities. It's also an opportunity to talk to students individually to check their wellbeing.

Reviewer: ```suggestion
Depending on your role, you might be responsible for a form group or registering students. During this time, pupils typically engage in morning activities. It's also an opportunity to talk to pupils individually to check their wellbeing.
```
what are morning activities?

#### 9:15am morning lessons 
The structure of the school day will depend on your school, but often there are 2 lessons taught in the morning followed by a break. In total, there may be 5 hour-long lessons in the day. Each class could be around 30 pupils and will have its own dynamics as well as unique needs.

$q-phil-secondary$

#### 11:15am morning break

> Teachers are sometimes responsible for break-time duties. This involves supervising pupils for their safety but it is an opportunity to interact with pupils outside of the classroom and build relationships. 

Reviewer: ```suggestion
Teachers are sometimes responsible for break-time duties. This involves supervising pupils for their safety but it's also an opportunity to interact with pupils outside of the classroom and build relationships. 
```

#### 11:35am lessons 

> After break, pupils return to lessons with renewed energy, which can be harnessed and channeled into their work. During this time, teachers usually teach two more lessons before lunch.

Reviewer: ```suggestion
After break, pupils return to lessons with renewed energy, which can be harnessed and channeled into their work. During this time, teachers usually teach 2 more lessons before lunch.
```

$q-jessica-science$

### Lunch

Lunch is an opportunity for teachers to have a well-earned break and go to the staff room to talk about how the morning has been. Many teachers also use this as an opportunity to get ahead on their marking. By tackling some books during this time, it can reduce the amount of marking needed to be done in the afternoon.

$q-mike-secondary$

### Afternoon

#### 2:15pm afternoon lessons 

> After lunch, there is often just one more lesson to teach. Schools sometimes work the timetable this way because after lunch pupils are generally more excitable making it harder to teach.  

Reviewer: ```suggestion
After lunch, there's often just one more lesson to teach. Schools sometimes work the timetable this way because after lunch pupils are generally more excitable making it harder to teach.  
```

#### 3:15pm pupils leave school
When pupils leave school the work continues. Teachers generally spend this time marking work and planning upcoming lessons.

Some teachers will also run an afterschool club such as a football team, drama sessions or computer club. 

$q-gavin-ict$

#### 5:00pm leave school

> Depending on how the teacher likes to split their work, they will leave school between 4:30pm to 6:00pm. Some teachers will take home books to mark or carry on planning once they are at home.

Reviewer: ```suggestion
Depending on how the teacher likes to split their work, they will leave school between 4:30pm and 6:00pm. Some teachers will take home books to mark or carry on planning once they're at home.
```

### Planning preparation and assessment time (PPA)

> Teachers are entitled 10% of their time off time table to plan work. Depending on the school, this could mean that you have a couple of lessons free next to each other or they could be spread across the week.

Reviewer: ```suggestion
Teachers are entitled to 10% of their time without lessons, so they can plan work. Depending on the school, this could mean that you have a couple of free periods next to each other or they could be spread across the week.
```
I found this a bit difficult to read/understand. Have tried to tweak.
